pkg/resourcecreator/google/bigquery/bigquery.go

```diff
@@ -0,0 +1,61 @@
+package google_bigquery
+
+import (
+	"fmt"
+	"strings"
+
+	google_bigquery_crd "github.com/nais/liberator/pkg/apis/bigquery.cnrm.cloud.google.com/v1beta1"
+	nais "github.com/nais/liberator/pkg/apis/nais.io/v1alpha1"
+	"github.com/nais/liberator/pkg/namegen"
+	"github.com/nais/naiserator/pkg/resourcecreator/google"
+	"github.com/nais/naiserator/pkg/resourcecreator/resource"
+	"github.com/nais/naiserator/pkg/util"
+	k8s_meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/validation"
+)
+
+func CreateBigQueryDataset(app *nais.Application, resourceOptions resource.Options, operations *resource.Operations, serviceAccountName string) error {
```

Comment:
Jeg ville nok endret metodenavnet til å bare være `Create` eller `CreateDataSet`, da vi er i modulen `google_bigquery`. For meg blir `bigquery` til redundant å skrive igjen.

Reply:
Done! Takker! =D